distributions/otelcol-k8s/manifest.yaml

```diff
@@ -0,0 +1,71 @@
+    import: github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.96.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.96.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.96.0
+
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.96.0
+  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/attributesprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatorateprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/filterprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbyattrsprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbytraceprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/metricstransformprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/remotetapprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/tailsamplingprocessor v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/transformprocessor v0.96.0
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/httpcheckreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver v0.96.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/journaldreceiver v0.96.0
```

Comment:
This is likely not that relevant for Kubernetes workloads

Reply:
The journald receiver is relevant, in that most K8s distributions use systemd for running the kubelet, so this receiver is the only way to get kubelet logs.

I think we should include hostmetrics, because it provides an Otel-exclusive way of collecting Node metrics, without relying on node-exporter and the like.

No strong opinions on the other two.

Reply:
See https://github.com/open-telemetry/opentelemetry-collector-releases/issues/357#issuecomment-1726424285

Reply:
> in that most K8s distributions use systemd for running the kubelet

When I think about a Kubernetes distribution, I think of a distribution to get telemetry from workloads. Getting kubelet telemetry would be similar to getting data from a regular process on bare metal (MySQL, email servers, ...). So, while I agree that systemd would be useful for getting telemetry out of kubelet, I don't think it would be useful for the workloads themselves.

Reply:
I would definitely argue we should also cover getting telemetry from kubernetes components themselves. These use cases go hand-in-hand and are in some cases even dependent on each other.

Reply:
@jpkrohling I agree with @swiatekm-sumo. The distribution's current criteria states: `All receivers must facilitate the collection of data that is generated by Kubernetes or services running in Kubernetes`. The kubelet logs themselves fall under `data that is generated by Kubernetes`.